CMakeLists.txt

```diff
@@ -335,12 +335,6 @@ set(RISCV   1)
 set(RISCV32 1)
 endif()
 
-if(WIN32 OR ARM OR PPC64LE OR PPC64 OR PPC)
```

Comment:
I think it would help simplify review and reduce merge confusion with other PRs to keep the OPT_FLAG_RELEASE setting in this location and remove the conditionals.

Reply:
Sorry for that, I thought it would be better to regroup somewhere relevant. Amending the commit seems too dangerous for now, any suggestion what should I do?

Reply:
I'd say just leave it for now until there's a consensus on the best resolution, as, setting aside jeffro256's comment, the proposed solutions appears to be 0-1 liners.